packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js

```diff
@@ -188,7 +188,7 @@ angular.module('App').controller(
     }
 
     getResumePrice(price) {
-      return price.value === 0 ? this.$translate.instant('price_free') : this.$translate.instant('price_ht_label', { price: price.text });
+      return price.value === 0 ? this.$translate.instant('price_free') : `${price.text} ${this.$translate.instant('price_ht_label')}`;
```

Comment:
Could you replace the translation by `{{price}} HT` instead ?

Reply:
Okay sure will do. I thought of using the translation which was already there :)

Reply:
Just make sure it's not used elsewhere and that you are not breaking anything 😄

Reply:
I tried it, the problem is the `"€"` that has to come between price` `and `HT`, even if i add a translation which will be only for `"HT"`, i will have to concatenate `price.text` and translation in the end

Reply:
I don't get it, please tell me:
- what happens when you have a `{{price}} HT` translation ? Isn't the `€` already in the `price`text ?

Reply:
Yes `€` is in the text but the problem is, its getting displayed as `&#8364;`. Its not getting sanitized.

Reply:
This works! thanks